iothub/service/src/Device.cs

```diff
@@ -109,9 +110,29 @@ public Device(string id)
         public virtual DeviceCapabilities Capabilities { get; set; }
 
         /// <summary>
-        /// Scope to which this device instance belongs to
+        /// The scope of the device. For edge devices, this is auto-generated and immutable. For leaf devices, set this to create child/parent
+        /// relationship.
         /// </summary>
+        /// <remarks>
+        /// For edge devices, the auto-generated device scope consists of a prefix "ms-azure-iot-edge://", the name of the parent edge device, a dash, and the
+        /// generation Id of the parent edge device (e.g. ms-azure-iot-edge://MyEdgeDevice-000000000000000000).
+        ///
+        /// For leaf devices, the value to set a parent edge device can be retrieved from the parent edge device's <see cref="Scope"/> property.
```

Comment:
When we say "leaf device" here, does that imply that the device is a non-edge device? 

Because if we are thinking about this in terms of the parent/child relationship tree, an edge device with no children would be a "leaf" of that tree.

Reply:
Yes, that is the common term for a non-edge device.